hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/handlers/NodeCreateHandler.java

```diff
@@ -118,6 +121,13 @@ public void handle(@NonNull final HandleContext handleContext) {
         recordBuilder.nodeID(node.nodeId());
     }
 
+    @NonNull
+    @Override
+    public Fees calculateFees(@NonNull final FeeContext feeContext) {
+        final var feeCalculator = feeContext.feeCalculator(SubType.DEFAULT);
```

Comment:
As @tinker-michaelj suggested, we need to add VPT to the calculate Fees. And then we can generate the `feeSchedules.json`

Reply:
I fixed it please recheck it I use the same way as we discuss